### PR TITLE
Add Xlayered-tx-pool to the config log printout

### PR DESCRIPTION
**Pull request by siladu**
Unrelated: clarify epoch length in javadoc

### CODE REVIEW

**Review by Gabriel-Trintinalia — APPROVED** [review]
LGTM

File: besu/src/main/java/org/hyperledger/besu/cli/ConfigurationOverviewBuilder.java [hunk 1]
```diff
@@ -237,7 +248,11 @@ public String build() {
     }
 
     if (isHighSpec) {
-      lines.add("High spec configuration enabled");
+      lines.add("Experimental High Spec configuration enabled");
```
Comment (gfukushima) [inline besu/src/main/java/org/hyperledger/besu/cli/ConfigurationOverviewBuilder.java:251]:
```suggestion
      lines.add("Experimental high spec configuration enabled");
```

File: besu/src/main/java/org/hyperledger/besu/cli/ConfigurationOverviewBuilder.java [hunk 2]
```diff
@@ -237,7 +248,11 @@ public String build() {
...
+    }
+
+    if (isLayeredTxPool) {
+      lines.add("Experimental Layered Transaction Pool configuration enabled");
```
Comment (gfukushima) [inline besu/src/main/java/org/hyperledger/besu/cli/ConfigurationOverviewBuilder.java:255]:
```suggestion
      lines.add("Experimental layered transaction pool configuration enabled");
```

File: besu/src/test/java/org/hyperledger/besu/cli/ConfigurationOverviewBuilderTest.java [hunk 3]
```diff
@@ -137,10 +137,22 @@ void setEngineApis() {
...
     final String highSpecEnabled = builder.build();
-    assertThat(highSpecEnabled).contains("High spec configuration enabled");
+    assertThat(highSpecEnabled).contains("Experimental High Spec configuration enabled");
+  }
+
+  @Test
+  void setLayeredTxPoolEnabled() {
+    final String layeredTxPoolDisabled = builder.build();
+    assertThat(layeredTxPoolDisabled)
+        .doesNotContain("Experimental Layered Transaction Pool configuration enabled");
```
Comment (gfukushima) [inline besu/src/test/java/org/hyperledger/besu/cli/ConfigurationOverviewBuilderTest.java:151]:
```suggestion
        .doesNotContain("Experimental layered transaction pool configuration enabled");
```

File: besu/src/test/java/org/hyperledger/besu/cli/ConfigurationOverviewBuilderTest.java [hunk 4]
```diff
@@ -137,10 +137,22 @@ void setEngineApis() {
...
+
+    builder.setLayeredTxPoolEnabled();
+    final String layeredTxPoolEnabled = builder.build();
+    assertThat(layeredTxPoolEnabled)
+        .contains("Experimental Layered Transaction Pool configuration enabled");
```
Comment (gfukushima) [inline besu/src/test/java/org/hyperledger/besu/cli/ConfigurationOverviewBuilderTest.java:156]:
```suggestion
        .contains("Experimental layered transaction pool configuration enabled");
```

**Review by gfukushima — APPROVED** [review]
lgtm, just added some suggestions to follow the current standard in startup logs

**Pull request merged**